test/stb_image_test.exs

```diff
@@ -58,4 +58,21 @@ defmodule StbImageTest do
     assert frames ==
              [<<180, 128, 70, 255, 171, 119>>, <<61, 255, 65, 143, 117, 255>>]
   end
+
+
+  test "save image png" do
+    read = StbImage.from_file(Path.join(__DIR__, "test.png"))
+    {:ok, img, {height, width, num_channels}, _, _} = read
+    :ok = StbImage.to_file(Path.join(__DIR__, "save_test.png"), "png", img, width, height, num_channels)
+    assert StbImage.from_file(Path.join(__DIR__, "save_test.png")) == read
```

Comment:
```suggestion
    save_at = Path.join(__DIR__, "save_test.png")
    :ok = StbImage.to_file(save_at, "png", img, width, height, num_channels)
    assert StbImage.from_file(save_at) == read
    File.rm!(save_at)
```

Maybe delete the generated file at the end? The same could be applied for the other test case. :)

Reply:
Nice catch. You can actually ask ExUnit to create a unique directory for you, this way you don't want to clean it up: https://hexdocs.pm/ex_unit/ExUnit.Case.html#module-tmp-dir